app/views/course/courses/index.html.slim

```diff
@@ -1,5 +1,10 @@
 = page_header do
-  = new_button([:course]) if can?(:create, Course)
+  - if can?(:create, Course.new)
+    = new_button([:course])
+  - elsif request = current_tenant.user_role_requests.find_by(user_id: current_user&.id)
```

Comment:
Instructor can create course? What if instructor wants to request to become admin?